model/qti/container/Container.php

```diff
@@ -222,6 +223,13 @@ public function fixNonvoidTags($html)
             // correctly closed element
             return trim(mb_substr($matches[0], 0, -2), 'UTF-8') . '></' . $matches[2] . '>';
         }, $html);
+
+        if ($content === null && (preg_last_error() === PREG_BACKTRACK_LIMIT_ERROR || preg_last_error() === PREG_RECURSION_LIMIT_ERROR)) {
```

Comment:
suggestion: instead of calling preg_last_error 2 times you could refactory it by adding another variable. You can also isolate this condition to facilitate [code readability](https://refactoring.guru/decompose-conditional):
```
$pregLastError = preg_last_error();

if ($this->isContentBroken())
....

private function isContentBroken(?$content, $pregLastError): bool
{
return  $content === null && ($pregLastError === PREG_BACKTRACK_LIMIT_ERROR || $pregLastError === PREG_RECURSION_LIMIT_ERROR)
}